lib/active_shipping/shipment_event.rb

```diff
@@ -2,14 +2,18 @@ module ActiveShipping
   class ShipmentEvent
     attr_reader :name, :time, :location, :message, :type_code
 
-    def initialize(name, time, location, message = nil, type_code = nil)
-      @name, @time, @location, @message, @type_code = name, time, location, message, type_code
+    def initialize(name, time, location, message = nil, type_code = nil, timezone_ambiguous = true)
```

Comment:
I'd say it should be up to the implementer of the parser to mark the timezone as unambiguous.  USPS simply copies across the time integers into a blank `Time.utc(...)`, other trackers we've implemented are also ambiguous.  Many other carriers are building their ShipmentEvents with `zoneless_time` as an argument.

So it feels better to err on the side of being ambiguous over not?

What about `nil` as the default?  Sucks for `ambiguous_zone?` to potentially return 3 values, but I don't feel super confident that the ambiguity is gone from the other carriers untouched in this PR